src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -4171,7 +4171,16 @@ and TcPseudoMemberSpec cenv newOk env synTypes tpenv synMemberSig m =
 /// Check a value specification, e.g. in a signature, interface declaration or a constraint
 and TcValSpec (cenv: cenv) env declKind newOk containerInfo memFlagsOpt thisTyOpt tpenv synValSig attrs =
     let g = cenv.g
-    let (SynValSig(ident=SynIdent(id,_); explicitTypeParams=ValTyparDecls (synTypars, synTyparConstraints, _); synType=ty; arity=valSynInfo; range=m)) = synValSig
+    let (SynValSig(ident=SynIdent(id,_); explicitTypeParams=ValTyparDecls (synTypars, synTyparConstraints, _); synType=ty; arity=valSynInfo; range=m; accessibility = access)) = synValSig
+    
+    match access with
+    | SynValSigAccess.Single(Some access)
+    | SynValSigAccess.GetSet(Some access, _, _)
+    | SynValSigAccess.GetSet(_, Some access, _)
+    | SynValSigAccess.GetSet(_, _, Some access) ->
+        error(Error(FSComp.SR.tcAccessModifiersNotAllowedInSRTPConstraint(), access.Range))
```

Comment:
If possible can we use `errorR` to not block the rest of the analysis ?

Reply:
it's `errorR` already